datastore/cloud-ndb/named_db.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import ndb
+
+
+class Book(ndb.Model):
+    title = ndb.StringProperty()
+
+
+# [START datastore_ndb_named_db]
+database = os.environ.get("DATASTORE_DATABASE", "prod-database")
+client = ndb.Client(database=database)
+
+
+# [END datastore_ndb_named_db]
```

Comment:
question: is the intent of this sample to only show client initiation? Also I don't fully understand why the client is initialised outside of the list_books() function if it's only used there.

Reply:
Yes, client initialization is the only thing that _really_ matters to this sample.